ios/sdk/WeexSDK/Sources/Protocol/WXAppMonitorProtocol.h

```diff
@@ -54,4 +54,8 @@
 
 - (void)commitAppMonitorAlarm:(NSString *)pageName monitorPoint:(NSString *)monitorPoint success:(BOOL)success errorCode:(NSString *)errorCode errorMsg:(NSString *)errorMsg arg:(NSString *)arg;
 
+@optional
+
+- (void)commitMonitorWithPage:(NSString *)pageName withPoint:(NSString *)monitorPoint withArgs:(NSDictionary *)args;
```

Comment:
maybe
` - (void)commitMonitorWithPage:(NSString *)pageName monitorPoint:(NSString *)monitorPoint args:(NSDictionary *)args   `
 is better?

Reply:
Yes, that is better